cpp/Doxyfile.in

```diff
@@ -836,7 +828,9 @@ EXCLUDE_SYMLINKS       = NO
 # exclude all test directories for example use the pattern */test/*
 
 EXCLUDE_PATTERNS       = columnWiseSort.h \
-                         smoblocksolve.h
+                         smoblocksolve.h \
```

Comment:
I think this and the above columnWiseSort are not needed?

Reply:
I think you are right. I'll remove those.